### DISCUSSION TITLE
Problem downloading htop - 401 for a blob

### DISCUSSION THREAD

**Discussion by PlanetWilson** [Homebrew/discussions#3375] [Everyday usage] [answered]
I get a 401 attempting to install htop via home-brew on an M1 MacBook Pro 14

➜  ~ brew install htop
==> Downloading https://ghcr.io/v2/homebrew/core/ncurses/manifests/6.3
Already downloaded: /Users/mark/Library/Caches/Homebrew/downloads/cf28a7d20961705db8268d50915a444b00a47753faf431b783c62ab3a09163e7--ncurses-6.3.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/ncurses/blobs/sha256:a1aabfa5d0fd9b2735b3d83d5378a447049190a34d85c3df9f2983beecbf83d5
Already downloaded: /Users/mark/Library/Caches/Homebrew/downloads/8ebbf3594fed8c0b8eb7d9e0a8fc015742445c1cbeeeb8969be785ca32fee246--ncurses--6.3.arm64_monterey.bottle.tar.gz
==> Downloading https://ghcr.io/v2/homebrew/core/htop/manifests/3.2.1
Already downloaded: /Users/mark/Library/Caches/Homebrew/downloads/8397a04eda5a455926f02cd65313a2db69d47e1ce3b1e15749b7f81eaa0e7a0e--htop-3.2.1.bottle_manifest.json
==> Downloading https://ghcr.io/v2/homebrew/core/htop/blobs/sha256:50735eb9e09ec8087f04640430d4bdac4941a0ce584dd1e52ec8ec8a58d743ed
curl: (22) The requested URL returned error: 401

Error: htop: Failed to download resource "htop"
Download failed: https://ghcr.io/v2/homebrew/core/htop/blobs/sha256:50735eb9e09ec8087f04640430d4bdac4941a0ce584dd1e52ec8ec8a58d743ed

**Comment by gromgit** [comment, marked as answer]
GitHub is having some issues right now: https://github.com/Homebrew/brew/issues/13460#issuecomment-1163042357
